### ISSUE TITLE
[19.04] Atlas - capabilities.json Duplicated key "name"

### ISSUE THREAD

**Issue by pierrejego** [georchestra/georchestra#2774]
I try to use Atlas from 19/04 version, 
when I try to launch the addons via mapfishapp, I got an error on  Altas webapps side
the atlas/print/capabilities.json return an error 

org.json.JSONException: Duplicate key "name"
	at org.json.JSONWriter.key(JSONWriter.java:219)
	at org.mapfish.print.attribute.PrimitiveAttribute.printClientConfig(PrimitiveAttribute.java:80)
	at org.mapfish.print.config.Template.printClientConfig(Template.java:111)
	at org.mapfish.print.config.Configuration.printClientConfig(Configuration.java:249)
	at org.mapfish.print.MapPrinter.printClientConfig(MapPrinter.java:73)
	at org.georchestra.atlas.CamelMapfishPrintComponent.printCapabilities(CamelMapfishPrintComponent.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.camel.component.bean.MethodInfo.invoke(MethodInfo.java:408)
	at org.apache.camel.component.bean.MethodInfo$1.doProceed(MethodInfo.java:279)
	at org.apache.camel.component.bean.MethodInfo$1.proceed(MethodInfo.java:252)
	at org.apache.camel.component.bean.BeanProcessor.process(BeanProcessor.java:177)
	at org.apache.camel.util.AsyncProcessorHelper.process(AsyncProcessorHelper.java:109)
	at org.apache.camel.component.bean.BeanProcessor.process(BeanProcessor.java:68)
	at org.apache.camel.component.bean.BeanProducer.process(BeanProducer.java:38)
	at org.apache.camel.processor.SendProcessor.process(SendProcessor.java:141)
	at org.apache.camel.management.InstrumentationProcessor.process(InstrumentationProcessor.java:77)
	at org.apache.camel.processor.RedeliveryErrorHandler.process(RedeliveryErrorHandler.java:460)
	at org.apache.camel.processor.CamelInternalProcessor.process(CamelInternalProcessor.java:190)
	at org.apache.camel.processor.Pipeline.process(Pipeline.java:121)
	at org.apache.camel.processor.Pipeline.process(Pipeline.java:83)
	at org.apache.camel.processor.CamelInternalProcessor.process(CamelInternalProcessor.java:190)
	at org.apache.camel.util.AsyncProcessorHelper.process(AsyncProcessorHelper.java:109)
	at org.apache.camel.processor.DelegateAsyncProcessor.process(DelegateAsyncProcessor.java:87)
	at org.apache.camel.http.common.CamelServlet.service(CamelServlet.java:143)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:197)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:526)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:678)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:860)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1587)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)

On Rennes Métropole instance result of https://portail.sig.rennesmetropole.fr/atlas/print/capabilities.json have duplication entries for name : 
{"layouts":[{"name":"A4 portrait","attributes":[{"name":"title","name":"title","type":"String"},{
but no errors in Atlas webapp.

I thought it was link to https://github.com/mapfish/mapfish-print/issues/621
but this issue was corrected on mapfishprint 3.20 version and we use 3.50 version in atlas webapps.

**Comment by fvanderbiest** [comment]
Atlas was tested on 18.06 indeed.
Never seen this bug before. Thanks for reporting

**Comment by pierrejego** [comment]
Hello I was looking at this bug, and I am a bit lost in mapfish-print version.

I fact in 19.04 webapp Atlas lib folder,  we find print-lib 2.50 version 
but when I look in configuration file, it should use 3.50 version if I m correct

There is no version set in atlas pom.xml
https://github.com/georchestra/georchestra/blob/19.04/atlas/pom.xml#L86
but in root
https://github.com/georchestra/georchestra/blob/19.04/pom.xml#L47
and 
https://github.com/georchestra/georchestra/blob/19.04/pom.xml#L332

I looked on maven and gradle repository (mapfish-print is build with gradle)
I could not manage to find print-lib 3.50 version, the lastest I found is 3.20.3

If I set the version in atlas pom.xml, It seems to work correctly. I said It seems, because it's the first time I used it.

I can not understand from where comes this 2.50 version
